modules/network/outputs.tf

```diff
@@ -0,0 +1,11 @@
+output "vrf_id" {
```

Comment:
This entire module needs a `terraform fmt -recursive`. Opting to do that in a separate PR to avoid noise.